apps/rewards/app/components/Content/MyRewards.js

```diff
@@ -143,6 +156,13 @@ MyRewards.propTypes = {
   myMetrics: PropTypes.arrayOf(PropTypes.object).isRequired,
 }
 
+const StyledIcon = ({ Icon }) => (
+  <Icon style={{
+    marginRight: GU * 1.5 + 'px',
+    marginBottom: GU / 4 + 'px',
```

Comment:
Originally, this was 11 and 2 pixels respectively. I made this change half-jokingly because it looks pretty awful and I'm not sure it makes sense to include `GU` in this case.

Reply:
I'm not sure either! I guess a way to think about it is: if `GU` changed to, say, `2`, would we want this to change accordingly? If `GU` changed to `10` or even `20`, would we want that? If so, let's use GU. If we want this to always be 2px, then let's skip GU here.

Reply:
OK, nice rule of thumb. I'm gonna skip it, then.